docs/docs/installation/rasa-pro/architecture-overview.mdx

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
I think we're missing a section here that mentions the feature in Rasa Pro, and links to their doc